crates/sui-framework/packages/bridge/sources/bridge.move

```diff
@@ -20,7 +20,7 @@ module bridge::bridge {
     use bridge::committee::{Self, BridgeCommittee};
     use bridge::message::{Self, BridgeMessage, BridgeMessageKey};
     use bridge::message_types;
-    use bridge::treasury::{Self, BridgeTreasury, token_id};
+    use bridge::treasury::{Self, BridgeTreasury};
```

Comment:
Changes in this file are some minor clean up unrelated to the new messages changes...